app/test_app/conftest.py

```diff
@@ -106,6 +106,17 @@ def create_transfer_account_user(test_client, init_database, create_organisation
     return user
 
 
+@pytest.fixture(scope='function')
+def create_transfer_account_user_function(test_client, init_database, create_organisation):
+    from server.utils.user import create_transfer_account_user
+    user = create_transfer_account_user(first_name='Transfer',
+                                        last_name='User 2',
+                                        phone=fake.msisdn(),
+                                        organisation=create_organisation)
+    db.session.commit()
+    return user
```

Comment:
Using `create_transfer_account_user` was causing problems in other tests due to the fixture being scoped to a `module`. This fixture is thus scoped to a `function`.

Reply:
is it viable to just scope the original `create_transfer_account_user` to a function? Broadly speaking i think it should be in most cases anyways...

Reply:
I think doing so caused some errors to occur in other tests - I will investigate